I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/main_loop.rpy", line 68, in script
    $ electricity_fsm.drainCharge()
  File "game/main_loop.rpy", line 68, in <module>
    $ electricity_fsm.drainCharge()
AttributeError: 'ElectricityFSM' object has no attribute 'drainCharge'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/main_loop.rpy", line 68, in script
    $ electricity_fsm.drainCharge()
  File "C:\Users\themi\OneDrive\Documents\Renpy\renpy-8.3.7-sdk\renpy\ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\themi\OneDrive\Documents\Renpy\renpy-8.3.7-sdk\renpy\python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/main_loop.rpy", line 68, in <module>
    $ electricity_fsm.drainCharge()
AttributeError: 'ElectricityFSM' object has no attribute 'drainCharge'

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
KoiCare 1.0
Wed Jun 18 02:06:44 2025
